Aleksandr Zorin/index.js

```diff
@@ -0,0 +1,111 @@
+function splitAndMerge(str, sp) {
+    var words = str.split(' ');
+    var newWords = [];
+    words.forEach(function (word) {
+        newWords.push(word.split('').join(sp))
+    });
+    return newWords.join(' ');
+}
+
+function convert(object) {
+    var arr = []
+    for (const key in object) {
+        arr.push([key, object[key]]);
+    }
+    return arr;
+}
+
+function toCamelCase(str) {
+    var splitString = str.split('-');
+    var newString = [];
+    if (splitString.length == 1) {
+        splitString = str.split('_');
+    }
+    for (var i = 0; i < splitString.length; i++) {
+        if (i == 0) {
+            newString.push(splitString[i])
+        } else {
+            newString.push(splitString[i].charAt(0).toUpperCase() + splitString[i].slice(1));
+        }
+    }
+    return newString.join('');
+}
+
+function reverse(str) {
+    var words = str.split(' ');
+    var newString = [];
+    words.forEach(function (word) {
+        newString.push(word.split('').reverse().join(''));
+    });
+    return newString.join(' ');
```

Comment:
@Zorin-Alex Works good. You can use `.map` method of array to handle all entries like `str.split(' ').map(  /* and here function from words.forEach */ )`